src/ralph/deployment/models.py

```diff
@@ -56,6 +56,27 @@ def autocomplete_str(self):
         return '<i>{}</i> {}'.format(self.get_type_display(), self.name)
 
 
+CONFIGURATION_HELP_TEXT = """
+All newline characters will be converted to Unix \\n newlines.
+You can use {{variables}} in the body.
+Available variables:
+<br>    - configuration_class_name
+<br>    - configuration_module
+<br>    - configuration_path
+<br>    - dc
+<br>    - deployment_id
+<br>    - domain
+<br>    - done_url
+<br>    - hostname
+<br>    - initrd
+<br>    - kernel
+<br>    - kickstart
+<br>    - ralph_instance
+<br>    - service_env
+<br>    - service_uid
+""".strip()
```

Comment:
maybe extend variables by example of rendered values like you did in documentation?